services-js/commissions-app/src/client/common/TextInput.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+export interface Props {
+  name: string;
+  placeholder: string;
+  required: boolean;
+}
+export default class FormWithElement extends React.Component<Props> {
+  render() {
+    return (
+      <div className="fs">
+        <div className="fs-c">
+          <div className="txt m-b300">
+            <label
+              htmlFor="FeedbackForm-${this.props.name}"
+              className="txt-l txt-l--sm"
+            >
+              {this.props.name}
+
+              {this.props.required && (
+                <span className="t--req" aria-hidden="true">
+                  {' '}
+                  Required
+                </span>
+              )}
+            </label>
+            <input
+              name={this.props.name}
+              className="txt-f txt-f--sm"
+              placeholder={this.props.placeholder}
+              id=""
```

Comment:
Need to fill this in to match `htmlFor` in the `<label>` to associate the two together.